### ISSUE TITLE
Redesign /contribute/ pages

### ISSUE THREAD

**Issue by hoosteeno** [mozilla/bedrock#6994]
## Description

The Get Involved link on www.mozilla.org/contribute takes visitors to an outdated experience, and now there is a more up-to-date experience at https://activate.mozilla.community. We should redirect the link to the new experience.

Out of scope:
Someday, in a different issue, we should...
* Survey visitors to this page
* Get rid of old content and urls no longer required
* Redirect old urls to the new experience

Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1537919

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Link to "Get Involved" is redirected from all locations in /contribute section of the site.

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark:
- [ ] This is an example of a risk. Use a checklist!
- [ ] This is an example of a risk.

#### Tasks
- [ ] This is an example of a task. Use a checklist!
- [ ] This is an example of a task.

**Comment by alexgibson** [comment]
@hoosteeno this seems like something that's relatively low-effort and has a high payoff (poor UX and technical debt of current pages). I'm curious, why does this needs a seven sheets?

**Comment by hoosteeno** [comment]
It may not need a 7 sheets; I think it needs some evaluation and recommendation by someone expert in the content area. So maybe this is an epic with a couple stories in it? 

* Figure out what pages go away; determine if there are any technical gotchas awaiting; specify the changes; make sure we aren't overlooking a stakeholder or dependency.
* Make the changes

**Comment by LucyeoH** [comment]
Hey! @hoosteeno  I can be the content expert here. For background the current page is the result of an experiment I ran in 2016 with the old Mozilla.org owner (Jenn Bertsch) to try and address the out of date and broken experience on /contribute. However, there was not enough bandwidth on the Mozilla.org team to continue to design experiments and update the content. Therefore like so many contribute pages before it the tasks became out of date. 

I propose 2 updates that are related but can happen separately for simplicity: 

1. Redirect the “Get Involved” button from /contribute/signup to activate.mozilla.community. _[Here](https://docs.google.com/document/d/1_0koIVpT2SIQq6mTmyeyRM4D_YcGp7dGxI9bcdChSQM/edit#) I have done a complete analysis of the existing activities and informed all relevant stakeholders (whose links we'd be removing)._

2. Modernize the look and feel of the existing /contribute page, remove outdated content/links. _I'm currently working on the full content analysis in that same doc._

**Comment by LucyeoH** [comment]
As per my convo with @hoosteeno  I think makes the most sense is to A) do the /getinvolved redirect now so we’re pointing people to a maintained site and not directing them at dead (“hazardous”) products like MozStumbler B) Implement the survey to get genuine data about the audience of the page on  /contribute so we can optimize the page content for audience.

**Comment by alexgibson** [comment]
So just be be clear on what the steps are here:

1. Redirect /contribute/getinvolved/ (and any sub-pages) to https://activate.mozilla.community/
2. Remove /contribute/studentambassadors/ and redirect back to /contribute/
3. Remove /contribute/friends/ and redirect back to /friends/
4. Implement a survey on /contribute.
5. Update /contribute/ with some more recent copy / messaging.

Is the above estimation accurate? If so, is there anything preventing us from doing items 1-3 without waiting on 4 & 5?

**Comment by LucyeoH** [comment]
Hey @alexgibson and @hoosteeno thanks so much for making this happen! We're so excited to see how this makes is easier for contributors to find the easiest ways to get involved. The current [BugHunter campaign ](https://events.mozilla.org/becomeabetatestingbughunter) on the Activate site just brought 11k beta testers and more than 150 bugs for Fenix and I'm sure having this linked will increase the  numbers even more.

I also filed a bug to get another link back to mozilla.org added to activate. @hoosteeno feel free to comment https://github.com/mozilla/activate.mozilla.community/issues/350

**Comment by hoosteeno** [comment]
Hi @LucyeoH ,

Regarding Alex's question in https://github.com/mozilla/bedrock/issues/6994#issuecomment-487942885, do you think the approach he suggests is adequate? If so we can get this prioritized.

**Comment by LucyeoH** [comment]
Hey!

Might be easier to do in a quick meeting (I'll propose a time for next week) but I think these generally make sense. 
 
My understanding of current status on 1-3. 
   * [x] Redirect /contribute/getinvolved/ (and any sub-pages) to https://activate.mozilla.community/
   * [x] Remove /contribute/studentambassadors/ and redirect back to /contribute/ -> Currently re-directs to campus.mozilla.community which is fine. Could also go back to /contribute. 
* [x] Remove /contribute/friends/  (already appears to be gone?)

Needs actions: 
* [ ] Implement a survey on /contribute.
* [ ] Update /contribute/ with some more recent copy / messaging.
* [ ] Review, cull, update all related links and pages.

**Comment by alexgibson** [comment]
Whilst we're discusing this, it would be great to also figure out a way to resolve this 3 year old bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1275323

**Comment by LucyeoH** [comment]
@alexgibson thanks for the reminder! I left a comment!

**Comment by couci** [comment]
Hello @alexgibson and @hoosteeno 
as per our discussion on Friday here is the information to change the e-mailing list in the contribute page:
The slug for that subscription is: about-mozilla, and the vendor ID is: Sub_About_Mozilla__c

Let me know if you need anything else

**Comment by alexgibson** [comment]
Hi @LucyeoH,

Just checking in on progress here. We've had a survey running on this page for nearly a couple of months. Does your team have any more concrete insights/ideas about what you would like to do with these pages, and what visitors find valulable?

From a roadmap point of view for moz.org, the /contribute pages are pretty soon to be the most out of date (brand wise) on the website. I'd like to try and press forward for us to try and do somethig here before the end of the year.

The options I see are:

1. Remove and redirect these pages to https://activate.mozilla.community/ (or some other preferred destination).
2. Update the branding on these pages to suit the modern look of moz.org (no content changes).
3. Update the content with something more relevant for today (a redesign).

Choice 1 here is the least effort, and choice 3 the most effort.

Open to other suggestions also

/cc @hoosteeno

**Comment by LucyeoH** [comment]
@alexgibson perfect timing! So we've hit 1000+ responses and are analyzing now. We can remove the survey whenever. 

I'm meeting with a team from OI tomorrow to talk about kicking-off option #3 a redesign using resources from OI. 

Questions for you, if we want to do #3: 
* Who from your team needs to be consulted/involved in a redesign?
* What is the timeline for completing this redesign? End of year? 

I can share a fleshed out proposal at the end of this week - let me know who I should include on it.

**Comment by alexgibson** [comment]
> Who from your team needs to be consulted/involved in a redesign?

I suggest looping in @hoosteeno as a first point of contact is the best bet here. Thanks

**Comment by LucyeoH** [comment]
@hoosteeno would love your thoughts about timeline & any additional stakeholders!

**Comment by alexgibson** [comment]
Myself, @hoosteeno and @LucyeoH met today to discuss plans for this page, and we agreed that we're going to redesign this page in Q1 2020, just as soon as boths teams have enough free resources. Until then, this page can sit as-is - it's not worth porting it to Protocol given that we've now comitted to redesigning early in Q1.

**Comment by alexgibson** [comment]
Ok, the /contribute/ page is now officially the last page on mozorg using the legacy Sandstone framework. All other Sandstone pages of the website have now been either ported to Protocol or decommissioned. As such, this issue is now the last remaining obstacle to removing that framework, which is a large and significant milestone on bedrock's technical roadmap.

We committed to redesigning this page in Q1 this year, which is fantastic. However, even if the redesigned page launches, say at the end of Q1, the old page (and hence framework) will still need to stay around after that date, probably until the end of Q2, whilst locales migrate to the new page. This issue has been open for nearly a year now, and I'm not sure I can justify keeping Sandstone around for 6 months for a single page, given all the other work we've achieved migrating other pages. With that in mind, I'd like to propose the following:

- We keep to our agreement of redesigning the page in Q1.
- Whilst that is happening, we do a simple port of the old page to Protocol. The page will look different, but content will stay the same. This can be used as a stop-gap whilst the redesign happens.

Doing the above means we can complete https://github.com/mozilla/bedrock/issues/8384, which removes a significant amount of technical debt.

Any objections @hoosteeno?
